Dockerfile

```diff
@@ -9,10 +9,9 @@ RUN cd src; ./build_release.sh
 FROM ubuntu:18.04
 ENV ZOKRATES_HOME=/home/zokrates/.zokrates
 
-COPY --from=build /build/src/scripts/install_libsnark_prerequisites.sh /tmp/
-
-RUN /tmp/install_libsnark_prerequisites.sh \
-&& useradd -u 1000 -m zokrates
+RUN apt-get update \
```

Comment:
Is it the intent to remove the libsnark dependencies here?

Reply:
yes, it only needs the `libgmp3-dev` to run, other dependencies used for the build are in the environment image